### PR TITLE
Create 20230928-core-team.md

### PR DESCRIPTION
**Pull request by AlexandraAAJ**
Adding the notes from our September Core team meeting.

### CODE REVIEW

**Comment by aleesteele** [comment]
Thanks @AlexandraAAJ - I've approved it for now, but I do think we will have to had a policy in place for handling people's information, as their data is being uploaded to Github, and the data retention policies are not clear for the platform.

Also flagging that I'll be writing a summary of this meeting for general use.

**Review by aleesteele — APPROVED** [review]
👍 Flagging for data policies and use under GDPR

File: governance/core-team/meeting-notes/20230928-core-team.md
Comment (aleesteele) [inline governance/core-team/meeting-notes/20230928-core-team.md:7]: It might be good to remove this zoom link as is now.

File: governance/core-team/meeting-notes/20230928-core-team.md
Comment (aleesteele) [inline governance/core-team/meeting-notes/20230928-core-team.md:37]:
```suggestion
   * Liz Hare, independent dog genetics/working dog researcher, working on accessibility for TTW
```

File: governance/core-team/meeting-notes/20230928-core-team.md
Comment (aleesteele) [inline governance/core-team/meeting-notes/20230928-core-team.md:25]: I.e. maybe we can anonymise these names? Will investigate and add more information here before we merge.

File: governance/core-team/meeting-notes/20230928-core-team.md
Comment (aleesteele) [inline governance/core-team/meeting-notes/20230928-core-team.md:99]: We can erase all the empty lines below?

File: governance/core-team/meeting-notes/20230928-core-team.md
Comment (aleesteele) [inline governance/core-team/meeting-notes/20230928-core-team.md:86]: Can erase this empty line?

File: governance/core-team/meeting-notes/20230928-core-team.md
Comment (aleesteele) [inline governance/core-team/meeting-notes/20230928-core-team.md:143]: Can erase empty lines here?

**Comment by kallewesterling** [comment]
I went through @aleesteele's comments on this PR and implemented a few of the "emptier" edits (like removing empty bullet points and fixing spelling mistakes).

@aleesteele's [comment about reformatting this section](https://github.com/the-turing-way/the-turing-way/pull/3299#discussion_r1360422019) still remains, however, @AlexandraAAJ:

>    * What's next? Core staff delivery group will keep on working on it and have a PR to share in a couple of week. Get feedback. 
>   * We probably will have another session in Nov

**Comment by kallewesterling** [comment]
Also to note that @aleesteele's comment about anonymising names of participants is still pending. Last update: @aleesteele 
to "investigate and add more information here before we merge."

**Comment by aleesteele** [comment]
Thank you @kallewesterling! This has been an ongoing discussion: with the collab cafe notes (#3553) we were advised to remove check-in question answers, and keep the notes minimally focused on 'meeting minutes'. For these core team calls (which are an exercise in project transparency), the answer is less clear and guidance with ongoing.

(Adding breadcrumbs to #3534 - where we are documenting this ongoing work!)

**Comment by bsipocz** [comment]
We are in the middle of reorganizing the repository  and as part of that work have moved some of the documents into a new repo. This has introcude code conflicts above, and therefore I am closing this pull request.

Could you please reopen this PR in the new `governance` repo: https://github.com/the-turing-way/governance?

**Comment by JimMadge** [comment]
This PR has been moved [to here](https://github.com/the-turing-way/governance/pull/4) :pray:.

**Pull request closed without merging**
